database_initializer.py

```diff
@@ -8,10 +8,11 @@
 playlist_columns = ['pid', 'name', 'collaborative', 'duration_ms']
 artist_columns = ['artist_uri', 'artist_name']
 track_columns = ['track_uri', 'track_name', 'album_uri']
+album_columns = ['album_uri','album_name']
 
 class DatabaseInitializer:
-    def __init__(self, hostname: str, dbname: str) -> None:
-        self.connection = psycopg2.connect(host=hostname, dbname=dbname)
+    def __init__(self, hostname: str, dbname: str, username: str, password: str) -> None:
```

Comment:
https://github.com/dsvinod90/spotify_dataset/pull/9/files?diff=split&w=0#r1120072783

Same as the comment in the link above. username and password should be optional fields. So if it is provided then it will be considered, otherwise there is no need for them.

Reply:
added another constructor with username and password

Reply:
We don't need a new constructor. Just add defaults to the params in the existing one.

Reply:
made appropriate changes.

Reply:
Did you remove the default params for username and password altogether?